src/joystick/linux/SDL_sysjoystick.c

```diff
@@ -182,6 +192,30 @@ static int GuessIsJoystick(int fd)
     return 0;
 }
 
+static int GuessIsSensor(int fd)
```

Comment:
There is a naming convention for the Sony sensor devices. I'm not sure if it's used by all controllers in the latest kernels, but it might be worth looking at as a hint here?

Reply:
I can add name from SDL_ShouldIgnoreGamepad() as an hint: " Motion Sensors", "Nintendo * IMU", " Accelerometer". There is also Wii extension control " IR", " Motion Plus", " Nunchuk", but i don't know if these are accelerometer and gyro. 
If there are others names let me know and I'll add them too.

Reply:
For now, Wii U accessories are ignored because they create 3 files while we read only from 1. And I don't know how to check the device type (gyro or accelerometer) other than by name because they report everything on `RX, RY, RZ` and `SDL_EVDEV_GuessDeviceClass()` returns `SDL_UDEV_DEVICE_ACCELEROMETER` for both.